script.js

```diff
@@ -1,12 +1,95 @@
-//You can edit ALL of the code here
+
+const allEpisodes = getAllEpisodes();
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+  
+  const oneEpisode = getOneEpisode();
+  allEpisodes.forEach(element => {makeDivForEpisode(element);
+  });
 }
 
-function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
-}
+// function makePageForEpisodes(episodeList) {
+//   const rootElem = document.getElementById("root");
+//   rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+// }
 
 window.onload = setup;
+
+const episodesDiv = document.querySelector("#episodsHolder");
+
+// function for displaying all episodes
+
+function makeDivForEpisode (episode) {
+  const divContainer = document.createElement("div");
+  divContainer.className = "episodeContainer";
+
+  let title = document.createElement("h3");
+  let episodeCode = `S${(String(episode.season).padStart(2,'0'))}E${String(episode.number).padStart(2,'0')}`;
+  title.innerText = `${episode.name} - ${episodeCode}`;
```

Comment:
```suggestion
const paddedSeason = episode.season.toString().padStart(2, '0');
const paddedEpisode = episode.number.toString().padStart(2, '0');
const episodeCode = `S${paddedSeason}E${paddedEpisode}`;

  title.innerText = `${episode.name} - ${episodeCode}`;
```
It's soooo hard to read :) better to divide it into 2 separate variables